R/plotAbundance.R

```diff
@@ -140,14 +140,13 @@ setGeneric("plotAbundance", signature = c("x"),
     }
 }
 
-
 #' @rdname plotAbundance
 #' @importFrom scater plotExpression
 #' @importFrom ggplot2 facet_wrap
 #' @export
 setMethod("plotAbundance", signature = c("SummarizedExperiment"),
     function(x,
-            rank = taxonomyRanks(x)[1],
+            rank = NULL,
```

Comment:
As it is not possible in practice to plot >50 features, I would keep the default value as taxonomyRanks(x)[1]